pkgs/development/python-modules/scooby/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  pname = "scooby";
+  version = "0.7.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-scD/uuAtepatt8Yn7b6PJMfSj9AT7iOy0HuVHyVvEhk=";
+  };
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
```

Comment:
Consider adding `pythonImportsCheck` especially since `PyPI` uploads often don't contain a test suite (`pytestCheckHook` errors if there are no tests but otherwise `checkPhase` is silently a no-op).